Scripts/Mend SCA/get-malicious-packages.sh

```diff
@@ -19,14 +30,14 @@ OUTPUT="[]"
 # Loop through each product in $PRODUCTS and get the uuid associated with it.
 for (( i=0; i<=$NUM_PRODUCTS-1; i++ ))
 do
-	CURRENT_PRODUCT_TOKEN=$(echo $PRODUCTS | jq -r ".[$i].uuid" )
-	echo "Getting Malicious Packages for Product $(($i+1))/$NUM_PRODUCTS: $CURRENT_PRODUCT_TOKEN"
-	
-	# Get all security alerts that have "MSC" in the name
-	CURRENT_OUTPUT=$(curl -s --location "$WS_API_URL/products/$CURRENT_PRODUCT_TOKEN/alerts/security?pageSize=10000&page=0&search=vulnerabilityName%3ALIKE%3AMSC" --header 'Content-Type: application/json' --header "Authorization: Bearer $JWT_TOKEN" | jq '.retVal')
-
-	# Add the current value to $OUTPUT
-	OUTPUT=$(echo "$OUTPUT $CURRENT_OUTPUT" | jq -s add)
+        CURRENT_PRODUCT_TOKEN=$(echo $PRODUCTS | jq -r ".[$i].uuid" )
+        echo "Getting Malicious Packages for Product $(($i+1))/$NUM_PRODUCTS: $CURRENT_PRODUCT_TOKEN"
+
+        # Get all security alerts that have "MSC" in the name
+        CURRENT_OUTPUT=$(curl -s --location "$MEND_URL/products/$CURRENT_PRODUCT_TOKEN/alerts/security?pageSize=10000&page=0&search=vulnerabilityName%3ALIKE%3AMSC" --header 'Content-Type: application/json' --header "Authorization: Bearer $JWT_TOKEN" | jq '.retVal')
+
+        # Add the current value to $OUTPUT
+        OUTPUT=$(echo "$OUTPUT $CURRENT_OUTPUT" | jq -s add)
 done
 
```

Comment:
Add a comment here letting the user know that raw output can be extracted at this point